stdlib/public/core/StringCreate.swift

```diff
@@ -19,33 +19,54 @@ internal func _allASCII(_ input: UnsafeBufferPointer<UInt8>) -> Bool {
   //
   // TODO(String performance): SIMD-ize
   //
-  let ptr = input.baseAddress._unsafelyUnwrappedUnchecked
-  var i = 0
-
   let count = input.count
-  let stride = MemoryLayout<UInt>.stride
-  let address = Int(bitPattern: ptr)
-
-  let wordASCIIMask = UInt(truncatingIfNeeded: 0x8080_8080_8080_8080 as UInt64)
-  let byteASCIIMask = UInt8(truncatingIfNeeded: wordASCIIMask)
+  var ptr = UnsafeRawPointer(input.baseAddress._unsafelyUnwrappedUnchecked)
+  var i = 0
```

Comment:
oops forgot to delete this unused variable. Doing direct pointer math instead of modifying i and using it as an offset reduced the generated code size by more than 1/3.